man/BSmooth.fstat.Rd

```diff
@@ -0,0 +1,59 @@
+\name{BSmooth.fstat}
+\alias{BSmooth.fstat}
+\title{
+    Compute F-statistics based on smoothed whole-genome bisulfite
+    sequencing data.
+}
+\description{
+    Compute F-statistics based on smoothed whole-genome bisulfite
+    sequencing data.
+}
+\usage{
+BSmooth.fstat(BSseq, design, contrasts, verbose = TRUE)
+}
+\arguments{
+    \item{BSseq}{An object of class \code{BSseq}.}
+    \item{design}{The design matrix of the bisulfite-sequencing experiment,
+        with rows corresponding to arrays and columns to coefficients to be
+        estimated.}
+    \item{contrasts}{Numeric matrix with rows corresponding to columns in
+        \code{design} and columns containing contrasts. May be a vector if
+        there is only one contrast.}
+    \item{verbose}{Should the function be verbose?}
+}
+\details{
+    \strong{TODO}
+}
+\value{
+    An object of class \linkS4class{BSseqStat}.
+}
+\author{
+    Kasper Daniel Hansen \email{khansen@jhsph.edu}
+}
+\seealso{
+    \code{\link{BSmooth}} for the input object and
+    \linkS4class{BSseq} for its class.
+    \linkS4class{BSseqTstat} describes the return class. This
+    function is likely to be followed by the use of
+    \code{\link{dmrFinder}}.
+}
+\examples{
+    if(require(bsseqData)) {
+        data(keepLoci.ex)
+        data(BS.cancer.ex.fit)
+        BS.cancer.ex.fit <- updateObject(BS.cancer.ex.fit)
+        ## Remember to subset the BSseq object, see vignette for explanation
+        ## TODO: Kind of a forced example
```

Comment:
All the examples for F-stat based functions are kind of forced since they use the 2-group cancer example data. Obviously it would be better to have a more comprehensive example, but I don't know that we have one simple enough to drop in at this late stage.